packages/cozy-scanner/src/DocumentTypeData.js

```diff
@@ -0,0 +1,293 @@
+    label: 'health_invoice'
+  },
+  {
+    id: '23',
+    classification: 'certificate',
+    subject: 'car',
+    label: 'registration'
+  },
+  {
+    id: '24',
+    classification: 'certificate',
+    subject: 'car',
+    label: 'car_insurance'
+  },
+  {
+    id: '25',
+    classification: 'invoicing',
+    categorie: 'car',
+    label: 'mechanic_invoice'
+  },
+  {
+    id: '26',
+    classification: 'invoicing',
+    categorie: 'car',
+    label: 'transport_invoice'
+  },
+  {
+    id: '27',
+    classification: 'invoicing',
+    categorie: 'phone',
+    label: 'phone_invoice'
+  },
+  {
+    id: '28',
+    classification: 'invoicing',
+    categorie: 'isp',
+    label: 'isp_invoice'
+  },
+  {
+    id: '29',
+    classification: 'invoicing',
+    categorie: 'energy',
+    label: 'energy_invoice'
+  },
+  {
+    id: '30',
+    classification: 'invoicing',
+    categorie: 'web',
+    label: 'web_service_invoice'
+  },
+  {
+    id: '31',
+    classification: 'contract',
+    subject: 'house',
+    label: 'lease'
+  },
+  {
+    id: '32',
+    classification: 'invoicing',
+    subClassification: 'payment_statement',
+    label: 'rent_receipt'
+  },
+  {
+    id: '33',
+    classification: 'contract',
+    subject: 'house',
+    label: 'house_insurance'
+  },
+  {
+    id: '34',
+    classification: 'tax_return',
+    label: 'tax_return'
+  },
+  {
+    id: '35',
+    classification: 'tax_notice',
+    label: 'tax_notice'
+  },
+  {
+    id: '36',
+    classification: 'tax_notice',
+    label: 'tax_timetable'
+  },
+  {
+    id: '37',
+    classification: 'invoicing',
+    label: 'invoices'
+  }
+]
+
+export const themes = [
+  {
+    id: 'theme1',
+    label: 'identity',
+    icon: 'people',
+    file_type_ids: ['1', '2', '3', '4', '5', '6']
+  },
+  {
+    id: 'theme2',
+    label: 'family',
+    icon: 'team',
+    file_type_ids: ['4', '5', '7', '8', '9', '10', '11']
+  },
+  {
+    id: 'theme3',
+    label: 'work_study',
+    icon: 'company',
+    file_type_ids: ['12', '13', '14', '15', '16', '17', '18']
+  },
+  {
+    id: 'theme4',
+    label: 'health',
+    icon: 'heart',
+    file_type_ids: ['19', '20', '21', '22']
+  },
+  {
+    id: 'theme5',
+    label: 'home',
+    icon: 'home',
+    file_type_ids: ['31', '32', '33', '27', '28', '29']
+  },
+  {
+    id: 'theme6',
+    label: 'transport',
+    icon: 'car',
+    file_type_ids: ['6', '23', '24', '25', '26']
+  },
+  {
+    id: 'theme7',
+    label: 'tax',
+    icon: 'bank',
+    file_type_ids: ['34', '35', '36']
+  },
+  {
+    id: 'theme8',
+    label: 'invoice',
+    icon: 'euro',
+    file_type_ids: ['27', '28', '29', '26', '30']
+  }
+]
+
+export const getItemsByCategory = ({ label }) => {
+  const items = []
+  themes.map(theme => {
+    if (theme.label === label) {
+      theme.file_type_ids.map(itemId => {
+        items.push(getItemById(itemId))
+      })
+    }
+  })
+  return items
+}
+
+export const getItemById = id => {
+  return items.find(item => item.id === id)
+}
```

Comment:
When calliing a method like that, I would not expect it to be O(n). Myabe you can build a indexed view of items ?

```
const itemsById = keyBy(items, item => item.id)
```